src/test/regress/expected/multi_mx_function_call_delegation.out

```diff
@@ -722,6 +722,29 @@ SET search_path TO multi_mx_function_call_delegation, public;
 select create_distributed_function('mx_call_func(int,int)');
 ERROR:  operation is not allowed on this node
 HINT:  Connect to the coordinator and run it again.
+-- show that we can call delegated function from worker nodes
+SET citus.log_remote_commands to true;
```

Comment:
Still, this function is NOT delegated. When it is delegated, you should see:
```SQL
CALL proc(1);
**DEBUG:  pushing down the procedure**
...
```

I'm not even sure  `delegated_function` is a function that can be pushed down. The smallest example: https://gist.github.com/onderkalaci/a65584b82bb8247f146c89ddaa2859d7

Would be nice to read: https://docs.citusdata.com/en/v10.2/develop/api_udf.html#create-distributed-function

We should have a function that is colocated with a distributed table.

If you see all the commands (e.g., iNSERT etc), it means that this procedure is not delegated, but each command is executed one-by-one.

Additional read to understand what does function delegation mean: https://techcommunity.microsoft.com/t5/azure-database-for-postgresql/making-postgres-stored-procedures-9x-faster-in-citus/ba-p/1906102